### ISSUE TITLE
Is there Codeception efficient workaround for waitForTextNotPresent method

### ISSUE THREAD

**Issue by markodemo1305** [Codeception/Codeception#5387]
I am trying to wait on page until some particular text dissapears.
Only text-related method is waitForText

In selenium, there is a method which does this automatically
http://www.software-testing-tutorials-automation.com/2013/10/how-to-use-selenium-ide-waitfortext-and.html

What I would need is to have waiter which would wait some amount of time until some test dissapears from page.

For example:

$I->waitForTextNotPresent('foo', 30);

Is there some efficient workround for this using for example waitForElementNotVisible action?

**Comment by edno** [comment]
There's no method like this one as part of `Webdriver` module. 
But you can implement a helper (or even submit a PR).
Example (based on [waitForText](https://github.com/Codeception/Codeception/blob/2.5/src/Codeception/Module/WebDriver.php#L2385)):
```php
function waitForTextNotPresent($text, $timeout = 10, $selector = null)
{
        $message = sprintf(
            'Waited for %d secs but text %s still not found',
            $timeout,
            Locator::humanReadableString($text)
        );

        if (!$selector) {
            $locator = WebDriverBy::xpath('//body');
        }
        else {
            $locator = $this->getLocator($selector);
        }

        $condition = WebDriverExpectedCondition::not(
            WebDriverExpectedCondition::textToBePresentInElement($locator, $text)
        );
        $this->webDriver->wait($timeout)->until($condition, $message);
}

**Comment by connorshea** [comment]
This'd be really nice to have in the core package 🙇
